app/scss/partials/_modal_exit_button.scss

```diff
@@ -19,9 +19,14 @@
 	width: 26px;
 	height: 26px;
 	border-radius: 15px;
-	border: solid 0.8px #325e97;
 	background-color: #f7f7f7;
 	@include transition(background-color 0.2s);
+	&.grey {
+		border: solid 0.8px #325e97;
+	}
```

Comment:
See my comment on `ModalExitButton` to make sure that this `grey` class gets applied when needed.

Reply:
Good idea, we can just apply the value of border here